Added instance groups; issue #57

MCLDG opened the pull request:
Resolves most of issue #57 with the exception of creating an ig pinned to an AZ. I'll update the issue accordingly and leave it open.

arun-gupta commented:
Since this is kops-specific, can we move this content to `cluster-install` instead?

MCLDG commented:
Yip, done. As the instance-group readme is so long, I've moved to sub-folder under cluster-install, and added a small section to cluster-install with a link.
